nino/endpoints.py

```diff
@@ -41,7 +65,18 @@ def read(self, large: bool = True, medium: bool = False):
         resp = self._client.get(url)
         return resp.content
 
-    def save(self, fp: str, *, large: bool = True, medium: bool = False):
+    def save(self, fp: str, *, large: bool = True, medium: bool = False) -> int:
```

Comment:
```suggestion
    def save(self, fp: Union[str, pathlib.Path], *, large: bool = True, medium: bool = False) -> int:
```
Should be safe to support [`pathlib.Path`](https://docs.python.org/3/library/pathlib.html#pathlib.Path) here without changing any code